assets/sass/trusted-login.scss

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
This is not a requirement for integration, merely the result of a brief exploration into how we could modify the styles on the Trusted Login screen if so desired.